object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/ObjectStoreClient.scala

```diff
@@ -33,20 +34,28 @@ class ObjectStoreClient[F[_], REQ_BODY, RES, RES_BODY](
   private val authorizationHeader: (String, String) =
     ("Authorization", config.authorizationToken)
 
+  private def expirationPeriodHeader(expirationPeriod: ObjectExpirationPeriod): (String, String) =
+    "X-EXPIRE-AFTER" -> expirationPeriod.toString
+
   private val url = s"${config.baseUrl}/object-store"
 
   /** Storing an object on an existing path will overwrite the previously stored object on that path. */
   def putObject[CONTENT](
     path: Path.File,
     content: CONTENT,
+    expirationPeriod: ObjectExpirationPeriod,
```

Comment:
is it worth defaulting to a `config.expirationPeriod`? I can see this being defined for the project.